src/components/TmBlock/TmBlock.vue

```diff
@@ -81,14 +82,30 @@ export default {
       default: () => [
         {
           tx: {
-            msg: {
-              inputs: [{ address: null }],
-              outputs: null
+            value: {
+              msg: [
+                {
+                  value: {
+                    inputs: [{ address: null }],
+                    outputs: null
+                  }
+                }
+              ]
             }
           }
         }
       ]
     },
+    candidates: {
```

Comment:
any reason we're using candidates as opposed to validators?

Reply:
I am still making the differentiation between validating candidates and not validating candidates. Shouldn't I?

Reply:
not sure. at some point we should probably sync with the SDK folks about all the words.

Reply:
Ok, changing it.